superset-frontend/images/icons/checkbox-half.svg

```diff
@@ -0,0 +1,22 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+ 
+    http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+<svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect x="4" y="4.5" width="16" height="16" rx="2" fill="#20A7C9"/>
```

Comment:
Any thoughts on adding these icons as an <Icon> component, so that we can simply return the correct SVG but style the whole thing with Emotion (and thus the theme colors) rather than hard-coded colors?

Reply:
I think we can still override these with emotion. Using SVGR they'll be react components. CSS colors override attributes set on the svg.